2-mandatory/3-financial-times.js

```diff
@@ -15,6 +16,11 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+    let arr = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
+      arr.push(allArticleTitles[i].split(" ").length);
+    }
+    return allArticleTitles[arr.indexOf(Math.min(...arr))];
```

Comment:
It can be more readable to break it into more lines. eg
```
const fewestWords = Math.min(...arr)
const indexOfArticle = arr.indexOf(fewestWords)
const articleTitle = allArticleTitles[indexOfArticle]
return articleTitle
```
Also, this works when all articles have a different number of words. What if two or more articles have the same number of words? arr.indexOf will find the first match.

Reply:
looks better now thank you 🥇